samples/rabbitmq/native-integration/Rabbit_9/NativeSender/Program.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Text;
+using RabbitMQ.Client;
+
+class Program
+{
+    static void Main()
+    {
+        var endpointName = "Samples.RabbitMQ.NativeIntegration.Sender";
+
+        Console.Title = endpointName;
+
+        var connectionFactory = new ConnectionFactory();
+
+        using var connection = connectionFactory.CreateConnection(endpointName);
+        using var channel = connection.CreateModel();
+
+        Console.WriteLine("Press enter to send a message");
+        Console.WriteLine("Press any key to exit");
+
+        while (true)
+        {
+            var key = Console.ReadKey();
+            Console.WriteLine();
+
+            if (key.Key != ConsoleKey.Enter)
+            {
+                return;
+            }
+
+            var properties = channel.CreateBasicProperties();
+
+            //not required but shows how to access basic properties in the receiving NServiceBus endpoint
+            properties.AppId = "MyNativeApp";
+
+            #region GenerateUniqueMessageId
+            var messageId = Guid.NewGuid().ToString();
+
+            properties.MessageId = messageId;
+
+            #endregion
+
+            #region CreateNativePayload
+            var payload = "<MyMessage><SomeProperty>Hello from native sender</SomeProperty></MyMessage>";
```

Comment:
Nitpick: We could change to json here to avoid using the xmlserializer

Reply:
Sure, but then we'd also have to update the sample readme which talks about the serializer used, and it didn't seem like it was worth the extra effort. We'd end up needing to version the readme since there isn't a single json serializer that's available for all the versions of the sample.

Is there a reason to avoid using the XML serializer that would make it worth doing all of that?